src/index.html

```diff
@@ -15,8 +15,31 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="dfsd">
+        <img
+          src="images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul>
+          <li class="link is-active">APPLE</li>
+          <li>SAMSUNG</li>
+          <li>SMARTPHONES</li>
+          <li class="blue">LAPTOPS & COMPUTERS</li>
+          <li>TABLETS</li>
+          <li>GADGETS</li>
+          <li>PHOTO</li>
+          <li>VIDEO</li>
+        </ul>
+      </nav>
+    </header>
```

Comment:
Add empty lines between multiline sibling blocks of HTML to add some 'air' and simplify reading. But don't add them between parent and child elements.